cmd/sum-coverage.go

```diff
@@ -66,6 +69,6 @@ var sumCoverageCmd = &cobra.Command{
 
 func init() {
 	sumCoverageCmd.Flags().BoolVarP(&summerOptions.Print, "print", "p", false, "prints to standard out only")
-	sumCoverageCmd.Flags().StringVarP(&summerOptions.Output, "output", "o", "codeclimate.json", "output path")
+	sumCoverageCmd.Flags().StringVarP(&summerOptions.Output, "output", "o", "coverage/codeclimate.json", "output path")
```

Comment:
I'm feeling like the string `"coverage/codeclimate.json"` should be made a constant somewhere to avoid these kinds of bugs. This is, by my count, the 3rd place that needed changing.